backend/endpoints/responses/rom.py

```diff
@@ -26,12 +26,24 @@ class RomSchema(BaseModel):
     name: Optional[str]
     slug: Optional[str]
     summary: Optional[str]
+    total_rating: Optional[str]
+    genres: list[dict]
+    franchises: list[dict]
+    collections: list[dict]
+    expansions: list[dict]
+    dlcs: list[dict]
+    remasters: list[dict]
+    remakes: list[dict]
+    expanded_games: list[dict]
+    companies: list[dict]
```

Comment:
is it possible to be more specific about the types of `dict`? like what `remasters` look like internally, if we can type that more explicitely with a TypedDict

Reply:
yeah that's not a bad idea, all those fields have the same structure but I can define some TypedDicts for them

Reply:
done